src/compositor/input/mod.rs

```diff
@@ -896,30 +901,30 @@ enum KeyAction {
     None,
 }
 
-fn process_keyboard_shortcut(modifiers: ModifiersState, keysym: Keysym) -> Option<KeyAction> {
-    if modifiers.ctrl && modifiers.alt && keysym == xkb::KEY_BackSpace
-        || modifiers.logo && keysym == xkb::KEY_q
+fn process_keyboard_shortcut(modifiers: ModifiersState, keysym: u32) -> Option<KeyAction> {
+    if modifiers.ctrl && modifiers.alt && keysym == KEY_BackSpace
+        || modifiers.logo && keysym == KEY_q
     {
         // ctrl+alt+backspace = quit
         // logo + q = quit
         Some(KeyAction::Quit)
```

Comment:
Hmm, debating in my head whether a `match` expression would be more idiomatic...

```rust
match (modifiers, keysym) {
    (ModifierState { ctrl: true, alt: true, .. }, KEY_q)
    | (ModifierState { logo: true, .. }, KEY_q) => Some(KeyAction::Quit),
    // ...
    _ => None,
}
```

What do you think?

I'm anticipating this kind of switch shenaniganery going anyway since the input system is probably gonna end up being more event listener-based anyway, since multiple different parts of Nadva will want key events:

```rust
// Potential future code.

// Get associated handler(s) for a key combination.
let handlers = self.keyboard_handlers.on(key_event);

for handler in handlers {
    if handler(key_event) == ControlFlow::Break(()) {
        break;
    }
}
```